src/services/PaymentService.ts

```diff
@@ -343,6 +343,7 @@ export class PaymentService {
       nextPayment: subscription.current_period_end,
       amountAfterCoupon: upcomingInvoice.total,
       priceId: price.id,
+      planId: price?.product as string
```

Comment:
But the price id does exist for lifetime products, is it in another different data structure @masterprog-cmd ?

Reply:
The price object is fetched from [subscriptions.list](https://docs.stripe.com/api/subscriptions/list). I don't know if the lifetime plan appears here but seems that this endpoint only get the active subscriptions. Moreover, the lifetime plans are handled from the controller:
```
if (user.lifetime) {
    response = { type: 'lifetime' };
} else {
    response = await paymentService.getUserSubscription(user.customerId);
}  
```
@sg-gs @apsantiso

Reply:
Ok. It seems that we can get the productId for one time purchases (prices) through the invoices. So we can get users most recent invoices and check if the payment is recurrent, the price is one_time type, etc. 

It would be good to be able to know if users are subscribed to X lifetime plan without making too much effort in a future, but the invoices are enough here.